lib/redis/distributed.rb

```diff
@@ -648,6 +648,15 @@ def sinterstore(destination, *keys)
       end
     end
 
+    # This command is similar to SINTER, but instead of returning the result set,
+    #   it returns just the cardinality of the result.
+    def sintercard(*keys, limit: nil)
+      keys.flatten!(1)
```

Comment:
If you support both `sintercard("key1", "key2")` and `sintercard(["key1", "key2"])` that should be tested.